core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -401,16 +401,24 @@ private[spark] class SparkSubmit extends Logging {
         // SPARK-33782 : This downloads all the files , jars , archiveFiles and pyfiles to current
         // working directory
         // SPARK-43540: add current working directory into driver classpath
+        // SPARK-47475: make download to driver optional so executors may fetch resource from remote
+        // url directly to avoid overwhelming driver network when resource is big and executor count
+        // is high
         val workingDirectory = "."
         childClasspath += workingDirectory
-        def downloadResourcesToCurrentDirectory(uris: String, isArchive: Boolean = false):
+
+        def downloadResourcesToCurrentDirectory(uris: String,
+                                                isArchive: Boolean = false,
```

Comment:
Could you take a look at `Apache Spark Coding Guideline` at this time?
- https://spark.apache.org/contributing.html
> For Scala code, Apache Spark follows the official [Scala style guide](http://docs.scala-lang.org/style/) and [Databricks Scala guide](https://github.com/databricks/scala-style-guide). The latter is preferred. To format Scala code, run ./dev/scalafmt prior to submitting a PR.

Reply:
For this case, the following is the rule.
> For method declarations, use 4 space indentation for their parameters

Reply:
Ran ` ./dev/scalafmt` as mentioned, it actually modified a lot of existing code style as well - is this expected? Or something I missed here?

Reply:
Ya, `scalafmt` is only recommended for newly added code. Please don't touch the existing code.

Reply:
Thanks for the tips @dongjoon-hyun! 
I tried running `./dev/scalafmt`, somehow a lot of existing code was refactored for style. Is this expected?